src/components/LinkChooser/LinkChooser.tsx

```diff
@@ -195,6 +196,16 @@ export const LinkChooser: FC<LinkChooserProps> = ({
         autoResize: true,
     });
 
+    useEffect(() => {
+        const { focusedKey } = state.selectionManager;
+        if (focusedKey && popoverRef.current && state.isOpen) {
+            const dropdownItem = popoverRef.current?.querySelector(`[data-key="${focusedKey}"]`);
+            if (dropdownItem) {
+                scrollIntoView(popoverRef.current, dropdownItem as HTMLElement);
+            }
+        }
+    }, [state.selectionManager.focusedKey, state.isOpen]);
```

Comment:
You could create variables for the dependencies and then reuse them inside the hook

Reply:
Fixed :)